java/client/src/main/java/glide/api/commands/GenericBaseCommands.java

```diff
@@ -280,6 +280,24 @@ CompletableFuture<Boolean> pexpireAt(
 
     // TODO move ScriptingAndFunctionsBaseCommands
     // TODO add note about routing on cluster client
+    /**
+     * Returns the absolute Unix timestamp (since January 1, 1970) at which the given <code>key</code>
```

Comment:
nit: should we maybe mention its in seconds here (even though I know it mentions that after the \@return)?
```suggestion
     * Returns the absolute Unix timestamp (since January 1, 1970) in seconds at which the given <code>key</code>
```